Documentation/gettingstarted/kubeproxy-free.rst

```diff
@@ -1158,6 +1158,20 @@ The current Cilium kube-proxy replacement mode can also be introspected through
     $ kubectl exec -it -n kube-system cilium-xxxxx -- cilium status | grep KubeProxyReplacement
     KubeProxyReplacement:   Strict	[eth0 (DR)]
 
+Graceful Termination
+********************
+
+Cilium's eBPF kube-proxy replacement supports graceful termination of service
+endpoints deletion. The feature requires at least Kubernetes version 1.20, and
+the feature gate ``EndpointSliceTerminatingCondition`` needs to be enabled. The
+feature can be disabled with the configuration option ``enable-k8s-terminating-endpoint``.
```

Comment:
nit, I would make a slight amendment here, for example:

[...] needs to be enabled. By default, the Cilium agent then detects such terminating Pod state. If desired, the feature can also be disabled with the configuration option ``--enable-k8s-terminating-endpoint=false``.